### PR TITLE
Om-6 done getOrder method

### PR DESCRIPTION
**Pull request by Bansi-rabadiya**
Om-6 done getOrder method

### CODE REVIEW

File: Controllers/HomeController.cs [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
+﻿using OrderManagement_DataLayer;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Web;
 using System.Web.Mvc;
+using OrderManagement.Mapper;
+using OrderManagement.Models;
+using System.Threading;
+using Order = OrderManagement.Models.Order;
```
Comment (kmvasani) [inline Controllers/HomeController.cs]: Remove this lines "using Order = OrderManagement.Models.Order;" and  "using OrderManagement_DataLayer;"

**Review by kmvasani — CHANGES_REQUESTED** [review]
Please change it

File: Controllers/HomeController.cs [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
 namespace OrderManagement.Controllers
 {
     public class HomeController : Controller
     {
         public ActionResult Index()
         {
-            return View();
+            try
+            {
+
```
Comment (PriyankaHirapara) [inline Controllers/HomeController.cs]: Please remove the extra spaces.

**Review by Bansi-rabadiya — COMMENTED** [review]
Remove extra space and lines

**Pull request merged**